platform/darwin/src/MGLLocationManager.m

```diff
@@ -49,6 +57,22 @@ - (CLActivityType)activityType {
     return self.locationManager.activityType;
 }
 
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 140000
+- (CLAccuracyAuthorization)accuracyAuthorization {
+    if (@available(iOS 14.0, *)) {
+        return self.locationManager.accuracyAuthorization;
+    } else {
+        return CLAccuracyAuthorizationFullAccuracy;
+    }
+}
+
+- (void)requestTemporaryFullAccuracyAuthorizationWithPurposeKey:(NSString *)purposeKey {
+    if (@available(iOS 14.0, *)) {
+        [self.locationManager requestTemporaryFullAccuracyAuthorizationWithPurposeKey:purposeKey];
```

Comment:
I'm guessing we need to add a new purpose key in the plist now?

Reply:
DO we need to make any changes to documentation to reflect this?

Reply:
The key comes from Apple. But yes we will have to modify our docs.